swagger.yml

```diff
@@ -148,7 +190,7 @@ paths:
   /users:
     get:
       security:
-        - basicAuth: []
+        - bearerAuth: []
       tags:
       - "users"
       summary: "Retunrs a list of users"
```

Comment:
Typo

Reply:
No. Only the login endpoint has basic auth.